bench/inline/dune

```diff
@@ -0,0 +1,13 @@
+(executable
```

Comment:
I'd call this directory "micro-benchmarks" rather than "inline".

Reply:
I settled for `micro`. It's already in the `bench` dir, so it's clear it's a benchmark.